introduction.md

```diff
@@ -1,2 +1,10 @@
 This is me. Nice to meet you :)
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?